packages/backend/src/config/Config.ts

```diff
@@ -64,6 +64,7 @@ export interface EthereumTvlConfig {
 }
 
 export interface ArbitrumTvlConfig {
+  readonly alchemyApiKey: string
```

Comment:
We now have two copies of `alchemyApiKey`, do we want to unify those entries now or after finishing feature stage?

Reply:
but aren't the keys different for arbitrum and ethereum ?

Reply:
They can be i.e if we want separate access monitoring for each provider, but it will work either way since in both cases we utilize same Alchemy

Reply:
Doesn't it require enabling 'arbitrum' in Alchemy dashboard? If so, one key is fine but please update the key on both staging and production before merging.

Reply:
I used Ethereum Mainnet api key for Arbitrum mainnet and it worked just fined. I'll double-check either way

Reply:
Alright, even if it works we don't know whether it is intentional or not, let's insert per-chain entries

Reply:
https://github.com/l2beat/l2beat/pull/1645/commits/65f2973fdda1aec9ad7638073a6defda8775216a